plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfigurator.java

```diff
@@ -301,7 +301,7 @@ public CNode describe(T instance, ConfigurationContext context) throws Exception
 
         // add constructor parameters
         for (int i = 0; i < parameters.length; i++) {
-            final Configurator c = context.lookup(attributes[i].getType());
+            context.lookup(attributes[i].getType());
```

Comment:
is this needed?

Reply:
I believe so, due to `context` being used on line 306
```
mapping.put(names[i], attributes[i].describe(instance, context));
```

Reply:
Coverage says no :laughing: